samples/stock/plotoptions/series-datagrouping-first-anchor/demo.js

```diff
@@ -0,0 +1,155 @@
+const hour = 3600 * 1000;
+const data = [
+    [0.2 * hour, 1], // first group
+    [1 * hour, 2], // first group
+    [1.3 * hour, 2], // first group
+    [2 * hour, 3], // second group
+    [3 * hour, 4], // second group
+    [4 * hour, 1],
+    [5 * hour, 2],
+    [6 * hour, 3],
+    [7 * hour, 4],
+    [8 * hour, 1],
+    [9 * hour, 2],
+    [10.3 * hour, 3], // last group
+    [11 * hour, 2], // last group
+    [11.7 * hour, 4] // last group
+];
+
+Highcharts.stockChart('container', {
+    chart: {
+        height: 800
+    },
+    xAxis: {
+        max: 12 * hour,
+        plotLines: [{
+            value: 0.2 * hour,
+            label: {
+                text: 'First point in group'
+            }
+        }, {
+            value: 1.3 * hour,
+            label: {
+                text: 'Last point in group'
+            }
+        }, {
+            value: 10.3 * hour,
+            label: {
+                text: 'First point in group'
+            }
+        }, {
+            value: 11.7 * hour,
+            label: {
+                text: 'Last point in group'
+            }
+        }],
+        plotBands: [{
+            from: 0,
+            to: 2 * hour,
+            color: '#dedede',
+            label: {
+                text: 'First group'
+            },
+            zIndex: -3
+        }, {
+            from: 10 * hour,
+            to: 12 * hour,
+            color: '#dedede',
+            label: {
+                text: 'Last group'
+            },
+            zIndex: -3
+        }]
+    },
+    yAxis: [{
+        height: '10%',
+        offset: 0,
+        title: {
+            text: 'Raw data'
+        }
+    }, {
+        height: '18%',
+        top: '10%',
+        offset: 0,
+        title: {
+            text: 'start'
+        }
+    }, {
+        height: '18%',
+        top: '28%',
+        offset: 0,
+        title: {
+            text: 'middle'
+        }
+    }, {
+        height: '20%',
+        top: '46%',
+        offset: 0,
+        title: {
+            text: 'end'
+        }
+    }, {
+        height: '20%',
+        top: '64%',
+        offset: 0,
+        title: {
+            text: 'firstPoint'
+        }
+    }, {
+        height: '20%',
+        top: '82%',
```

Comment:
Please keep the top+height <= 100% :) 

Otherwise, we get overflowing bands here:

![Zrzut ekranu 2021-04-14 o 13 07 14](https://user-images.githubusercontent.com/1453926/114700883-642c3600-9d22-11eb-9dc2-dc506a513efa.png)

PS: Cool demo!